scripts/convert_svd_to_diffusers.py

```diff
@@ -39,56 +58,58 @@ def create_unet_diffusers_config(original_config, image_size: int, controlnet=Fa
+    if unet_params["transformer_depth"] is not None:
         transformer_layers_per_block = (
-            unet_params.transformer_depth
-            if isinstance(unet_params.transformer_depth, int)
-            else list(unet_params.transformer_depth)
+            unet_params["transformer_depth"]
+            if isinstance(unet_params["transformer_depth"], int)
+            else list(unet_params["transformer_depth"])
         )
     else:
         transformer_layers_per_block = 1
 
-    vae_scale_factor = 2 ** (len(vae_params.ch_mult) - 1)
+    vae_scale_factor = 2 ** (len(vae_params["ch_mult"]) - 1)
 
-    head_dim = unet_params.num_heads if "num_heads" in unet_params else None
+    head_dim = unet_params["num_heads"] if "num_heads" in unet_params else None
     use_linear_projection = (
-        unet_params.use_linear_in_transformer if "use_linear_in_transformer" in unet_params else False
+        unet_params["use_linear_in_transformer"] if "use_linear_in_transformer" in unet_params else False
     )
     if use_linear_projection:
         # stable diffusion 2-base-512 and 2-768
         if head_dim is None:
-            head_dim_mult = unet_params.model_channels // unet_params.num_head_channels
-            head_dim = [head_dim_mult * c for c in list(unet_params.channel_mult)]
+            head_dim_mult = unet_params["model_channels"] // unet_params["num_head_channels"]
+            head_dim = [head_dim_mult * c for c in list(unet_params["channel_mult"])]
 
     class_embed_type = None
     addition_embed_type = None
     addition_time_embed_dim = None
     projection_class_embeddings_input_dim = None
     context_dim = None
 
-    if unet_params.context_dim is not None:
+    if unet_params["context_dim"] is not None:
         context_dim = (
-            unet_params.context_dim if isinstance(unet_params.context_dim, int) else unet_params.context_dim[0]
+            unet_params["context_dim"]
+            if isinstance(unet_params["context_dim"], int)
+            else unet_params["context_dim"][0]
         )
 
     if "num_classes" in unet_params:
-        if unet_params.num_classes == "sequential":
+        if unet_params["num_classes"] == "sequential":
             addition_time_embed_dim = 256
             assert "adm_in_channels" in unet_params
-            projection_class_embeddings_input_dim = unet_params.adm_in_channels
+            projection_class_embeddings_input_dim = unet_params["adm_in_channels"]
 
     config = {
         "sample_size": image_size // vae_scale_factor,
-        "in_channels": unet_params.in_channels,
+        "in_channels": unet_params["in_channels"],
         "down_block_types": tuple(down_block_types),
         "block_out_channels": tuple(block_out_channels),
-        "layers_per_block": unet_params.num_res_blocks,
+        "layers_per_block": unet_params["num_res_blocks"],
         "cross_attention_dim": context_dim,
         "attention_head_dim": head_dim,
```

Comment:
@patil-suraj I have buttoned up the code a bit. 

Now when I am running:

```bash
python convert_svd_to_diffusers.py --checkpoint_path=/raid/.cache/huggingface/models--stabilityai--stable-video-diffusion-img2vid-xt-1-1/snapshots/bfaf882a74971cae6fe4c22935110030db21fae9/svd_xt_1_1.safetensors --config_file=svd_xt_image_decoder.yaml --dump_path=svd_v1_1
```

It's running into:

```
Traceback (most recent call last):
  File "/home/sayak/collabs/diffusers/scripts/convert_svd_to_diffusers.py", line 769, in <module>
    unet = UNetSpatioTemporalConditionModel(**unet_config)
  File "/home/sayak/diffusers/src/diffusers/configuration_utils.py", line 649, in inner_init
    init(self, *args, **init_kwargs)
TypeError: UNetSpatioTemporalConditionModel.__init__() got an unexpected keyword argument 'attention_head_dim'
```

Any suggestions?